Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -1557,6 +1558,21 @@ void getSupportedTypes(HashSet<String>& types) const final
     return LogMediaSource;
 }
 
+void MediaPlayerPrivateMediaSourceAVFObjC::setShouldMaintainAspectRatio(bool shouldMaintainAspectRatio)
+{
+    m_shouldMaintainAspectRatio = shouldMaintainAspectRatio;
+
+    if (m_sampleBufferDisplayLayer) {
```

Comment:
I'd bail out early if `m_shouldMaintainAspectRatio == shouldMaintainAspectRatio `, and early return if `! m_sampleBufferDisplayLayer`.

Reply:
ok